spec/expression.dd

```diff
@@ -1749,8 +1746,22 @@ $(GNAME FunctionLiteralBody):
         }
         -------------
 
-    $(P and the following where the return type $(D int) and
-        $(D function)/$(D delegate) are inferred:)
+    $(P The use of `ref` declares the return value is done by reference:)
```

Comment:
"is done by" should be "is returned by".

Alternatively, "the returned value is passed by reference"